client/src/themes/theme.js

```diff
@@ -2,13 +2,20 @@ import { createMuiTheme } from "@material-ui/core";
 
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Roboto"',
-    fontSize: 12,
+    fontFamily: '"Open Sans"',
+    allVariants: {
+      color: "#5E6676",
+    },
     h1: {
-      // could customize the h1 variant as well
-    }
+      fontSize: "40px",
```

Comment:
For font sizes, let's not use pixels, instead you can just write fontSize: 40 (see https://material-ui.com/customization/typography/#font-size)